src/components/UI/Modal/Modal.module.scss

```diff
@@ -0,0 +1,36 @@
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vw;
+    min-width: 50%;
+    z-index: 2;
+}
+
+.modalMain {
+    margin-top: 50%;
```

Comment:
I suggest using the position absolute to center the application. Similarly how it's done [here](https://github.com/zero-to-mastery/mappypals/blob/master/src/pages/Home/AddFriendForm/InvitationSent.css)